content/02.main-text.md

```diff
@@ -114,6 +114,10 @@ As of {{creation_date_pretty}}, the Deep Review repository accumulated {{total_c
 The notebook to generate this figure can be [interactively launched using Binder](https://mybinder.org/v2/gh/greenelab/meta-review/binder?filepath=analyses/deep-review-contrib/02.contrib-viz.ipynb) [@doi:10.25080/Majora-4af1f417-011], enabling users to explore alternative visualizations or analyses of the source data.
 ](images/deep-review-contribution-ridge.svg){#fig:contrib width="100%" .white}
 
+GitHub issues and even pull requests can also be used for formal peer review by independent or journal-selected reviewers.
```

Comment:
"and even pull requests": are you envisioning a reviewer opens PRs. This would make sense in terms of typos, but it gets complicated when the reviewer becomes to contribute meaningful changes that qualify for authorship.